editor/editor_node.cpp

```diff
@@ -6032,10 +6032,10 @@ EditorNode::EditorNode() {
 	EDITOR_DEF("interface/inspector/horizontal_vector_types_editing", true);
 	EDITOR_DEF("interface/inspector/open_resources_in_current_inspector", true);
 	EDITOR_DEF("interface/inspector/resources_to_open_in_new_inspector", "Script,MeshLibrary");
-	EDITOR_DEF("interface/inspector/default_color_picker_mode", 0);
+	EDITOR_DEF("interface/inspector/default_color_picker_mode", 1);
```

Comment:
Using OKHSL as the default might not be a good idea, as it's a restrictive color space by design.

Reply:
Will update.